sycl/doc/SYCLInstrumentationUsingXPTI.md

```diff
@@ -227,16 +227,23 @@ by the SYCL runtime.
 
 ## SYCL Stream `"sycl.pi"` Notification Signatures
 
-| Trace Point Type | Parameter Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | Metadata |
-| ---------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| Trace Point Type | Parameter Description | Metadata |
+| :--------------: | :-------------------- | :------- |
```

Comment:
Looks like clang-format will add additional "---" to match field width, something to look out for. Thanks!

Reply:
I think `clang-format` is applicable for C-based programming languages, so Markdown documents should be ignored.